src/app/campaign/store/selectors/campaign-flight.selectors.ts

```diff
@@ -69,3 +76,85 @@ export const selectRoutedCampaignFlightDocs = createSelector(
     return { campaignDoc: campaign.doc, flightDoc: flight.doc };
   }
 );
+
+export const selectCampaignFlightInventoryReportData = createSelector(
+  selectCampaign,
+  selectAllInventory,
+  selectAllAdvertisers,
+  selectRemoteFlightsOrderByContractStartAt,
```

Comment:
using the remoteFlight, can't go back now...

Honestly not sure about this, but I felt like it would be surprising if a report had unsaved data. However, we overwrite the inventory flight days with the preview data instead of storing the preivew separately. The actuals shouldn't change in the preview, but it could still be weird/off.

Reply:
Agreed - it would be surprising for anything unsaved to show up.  Maybe that's an argument to store both local/preview and remote/persisted "flight-days"?  Such an edge case though ... I don't think it's worth addressing yet.